python/ql/src/semmle/python/dataflow/new/TypeTracker.qll

```diff
@@ -87,11 +87,26 @@ private predicate typePreservingStep(Node nodeFrom, Node nodeTo) {
   nodeFrom = nodeTo.(PostUpdateNode).getPreUpdateNode()
 }
 
+/**
+ * Gets a callable for the call where `nodeFrom` is used as the `i`'th argument.
```

Comment:
```suggestion
 * Gets the callable for the call where `nodeFrom` is used as the `i`'th argument.
```
I hope we only have one callable for each call, otherwise I think we confuse the data-flow library.

Reply:
ok, I wasn't sure, and went with the safer option.

Reply:
So I actually tried this out, and it turns out that it _can_ have multiple results. See commit ab856d6c01a388dbf07b45ff10300cf9102d4d44.

Reply:
Ah, of course, I did feel that I had not been thinking long enough about this. This must be ok, since we cannot always know which callable to choose. I think it will also be alleviated by CFG splitting, but in general it will have to be ok.